hardware/OpenWeatherMap.cpp

```diff
@@ -145,6 +145,8 @@ void COpenWeatherMap::GetMeterDetails()
 	{
 		bool bret;
 		std::string szURL = sURL.str();
+		HTTPClient::SetConnectionTimeout(5);
+		HTTPClient::SetTimeout(5);
```

Comment:
GSM or Modem 33600 is faster, and don't need 90 sec for get a few bytes of json.
I may increase connectiontimeout